devenv/doctor.py

```diff
@@ -106,6 +106,28 @@ def load_checks(repo: Repository, match_tags: set[str]) -> List[Check]:
     return checks
 
 
+def load_builtin_checks(match_tags: set[str]) -> List[Check]:
+    """
+    Loads builtin checks.
+    Optionally filter by tags.
+    If a check doesn't have the required attributes, skip it.
+    """
+    checks: list[Check] = []
+    match_tags.add("builtin")
+
+    for _, module_name, _ in walk_packages(devenv.checks.__path__):
+        module = __import__(f"devenv.checks.{module_name}", fromlist=["_trash"])
+        try:
+            check = Check(module)
+        except AssertionError as e:
```

Comment:
`except AssertionError` is usually a smell -- this should be ValueError or TypeError probably?